go/pkg/cas/upload.go

```diff
@@ -68,12 +70,17 @@ func (c *Client) Upload(ctx context.Context, inputC <-chan *UploadInput) (stats
 		eg:     eg,
 
 		fsSem: semaphore.NewWeighted(int64(c.FSConcurrency)),
+
+		semBatches: semaphore.NewWeighted(int64(c.BatchUpdateBlobsConcurrency)),
```

Comment:
yes, it is. If there are multiple callers to `Upload`, then they would have separate semaphores. Are you suggesting to move `semBatches` to `Client`?

Reply:
+1, the current `client` package sets a client wide batch concurrency limit as well.

Reply:
done, and renamed to `semBatchUpdateBlobs` since now it is at the `Client` level and "batches" can mean anything.

Also add a TODO to move `semFS` for the same reasons.